Hazel/src/Hazel/Core/Log.cpp

```diff
@@ -1,21 +1,36 @@
 #include "hzpch.h"
-#include "Hazel/Core/Log.h"
+//#include "Hazel/Core/Log.h"		included in hzpch.h already
 
-#include <spdlog/sinks/stdout_color_sinks.h>
+#include <spdlog/sinks/basic_file_sink.h>
 
 namespace Hazel {
 
 	Ref<spdlog::logger> Log::s_CoreLogger;
 	Ref<spdlog::logger> Log::s_ClientLogger;
+	Ref<spdlog::sinks::stdout_color_sink_mt> Log::s_ConsoleSink;
 
 	void Log::Init()
 	{
+		s_ConsoleSink = std::make_shared<spdlog::sinks::stdout_color_sink_mt>();
+
+		std::vector<spdlog::sink_ptr> coreSinks;
+		coreSinks.emplace_back(s_ConsoleSink); // VS debug console
```

Comment:
This isn't necessarily required, but it won't always be the Visual Studio debug console, which somewhat makes this comment useless.